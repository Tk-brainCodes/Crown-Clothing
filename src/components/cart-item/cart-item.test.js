import {shallow} from 'enzyme';
import React from 'react';
import CartItem from './cart-item.component';

it("expects to render CartItem", () => {
    const mockItems = [
        {
            imageUrl: 'https://jacket.png',
            name: 'black jacket',
            price: 50,
            quantity: 1
        }
    ]
    expect(shallow(<CartItem item={mockItems}/>)).toMatchSnapshot();
})