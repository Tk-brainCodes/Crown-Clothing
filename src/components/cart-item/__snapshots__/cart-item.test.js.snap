// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render CartItem 1`] = `ShallowWrapper {}`;
